test/e2e/specs/a8c-for-agencies/signup__smoke.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * @group calypso-pr
+ */
+
+import {
+	TestAccount,
+	getTestAccountByFeature,
+	envToFeatureKey,
+	envVariables,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+
+declare const browser: Browser;
+
+const A4A_URL = 'https://agencies.automattic.com';
+
+/**
+ * Verifies the A4A > Signup page loads
+ */
+describe( 'A4A > Signup: Smoke Test', function () {
+	let page: Page;
+	const accountName = getTestAccountByFeature( envToFeatureKey( envVariables ) );
+	const testAccount = new TestAccount( accountName );
+
+	beforeAll( async function () {
+		page = await browser.newPage();
+		await testAccount.authenticate( page );
+	} );
+
+	it( 'Navigate to A4A > Signup', async function () {
+		await page.goto( `${ A4A_URL }/signup` );
+
+		// Enter first name
+		const firstName = 'John';
+		await page.getByLabel( 'First name' ).fill( firstName );
+
+		// Enter last name
+		const lastName = 'Doe';
+		await page.getByLabel( 'Last name' ).fill( lastName );
+
+		// Enter the agency name
+		const agencyName = 'Agency Name';
+		await page.getByLabel( 'Agency name' ).fill( agencyName );
+
+		// Enter the business URL
+		const businessURL = 'https://example.com';
+		await page.getByLabel( 'Business URL' ).fill( businessURL );
+
+		// Select the user type to site owner
+		await page
+			.getByRole( 'combobox', { name: 'How would you describe yourself?' } )
+			.selectOption( { value: 'site_owner' } );
```

Comment:
Here, it will be better to use `label` instead of `value`

```suggestion
			.selectOption( { label: `I do not work at an agency. I'm a site owner.` } );
```

Reply:
Thanks! Fixed it here: https://github.com/Automattic/wp-calypso/pull/94747/commits/1de50a133dca7b07d5523aad70e58ef77c8c4389